source/administration/changelog.md

```diff
@@ -4,6 +4,148 @@ This changelog summarizes updates to [Mattermost Team Edition](http://www.matter
 
 Also see [changelog in progress](http://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.16 - [Feature Release](https://docs.mattermost.com/process/release-faq.html#release-overview)
+
+**Release Day: 2019-10-16**
+
+### Compatibility
+
+#### Removed and Deprecated Features
+
+ - We removed support for Internet Explorer (IE11). Learn more in our [forum post](https://forum.mattermost.org/t/mattermost-is-dropping-support-for-internet-explorer-ie11-in-v5-16/7575).
+ 
+### Breaking Changes
+ - 
+
+### Highlights
+
+#### Guest Accounts
+ - Provides a controlled and secure method for users outside of an organization to collaborate with their organization without allowing the guest to access proprietary or confidential information.
+
+#### Plugin Marketplace
+ - The integrations marketplace is built into the product and gives system administrators the ability to discover and install Mattermost plugins that are compatible with the server version you are running.  
+
+#### Improved team/channel management
```

Comment:
This was added in 5.14 https://docs.mattermost.com/administration/changelog.html#system-console-tools-to-manage-ldap-groups-within-teams-and-channels-ee.  
We could replace with:  
**Improved user management** - System Admins can view a user's team memberships and add a user to additional teams from the System Console without having to be a member of the team.